### ISSUE TITLE
Not possible to embed a auto-wired controller in Twig

### ISSUE THREAD

**Issue by floriansemm** [symfony/symfony#23083]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | no
| Symfony version  | 3.3.0

I don't know if it is a bug or just not yet implemented.

My Controller contains this action

```php
    public function dashboardAction(ActivityStreamManagerInterface $activityStreamManager, Request $request)
    {
        // more code
    }
```

and is embedded in my template like this:

```
{{ render(controller('AppBundle:ActivityStream:dashboard')) }}
```

When I try to render the template I got an error

`An exception has been thrown during the rendering of a template ("Controller "AppBundle\Controller\ActivityStreamController::dashboardAction()" requires that you provide a value for the "$activityStreamManager" argument. Either the argument is nullable and no null value has been provided, no default value has been provided or because there is a non optional argument after this one.").`

**Comment by nicolas-grekas** [comment]
This should have been fixed in 3.3.2, can you please try and confirm?

**Comment by floriansemm** [comment]
Sorry I missed this update. It is fixed.
